bin/oli/README.md

```diff
@@ -72,17 +72,17 @@ For different services, you could find the configuration keys in the correspondi
 ### Example: use `oli` to upload file to AWS S3
 
 ```text
-$ oli cp ./update-ecs-loadbalancer.json s3://update-ecs-loadbalancer.json
-$ oli ls s3://                                                           
+$ oli cp ./update-ecs-loadbalancer.json s3:/update-ecs-loadbalancer.json
+$ oli ls s3:/
 fleet.png
 update-ecs-loadbalancer.json
 ```
 
 ### Example: use `oli` copy file from S3 to R2
 
 ```text
-$ oli cp s3://fleet.png r2://fleet.png
-$ oli ls r2://
+$ oli cp s3:/fleet.png r2:/fleet.png
```

Comment:
That's more like a bug to me :sob:

Reply:
Oh, I convinced myself this is by design 🙈